.circleci/config.yml

```
@@ -76,7 +76,7 @@ commands:
     description: "Install riot"
     steps:
       # Make sure we install and run riot on Python 3
-      - run: pip3 install riot==0.17.3
+      - run: pip3 install git+https://github.com/DataDog/riot@fix/dev-package-prefix
```

Merge this suggestion once this fix is approved, merged and released: https://github.com/DataDog/riot/pull/209
```suggestion
      - run: pip3 install riot==0.17.4
```